lib/vsc/ldap/utils.py

```diff
@@ -29,8 +29,10 @@
 @author: Stijn De Weirdt
 @author: Wouter Depypere
 """
-import ldap
 import logging
+from six import add_metaclass
```

Comment:
We have the `py2vs3` module in `vsc-base` which is supposed to prevent us from depending on stuff like `six` (which isn't listed in `setup.py` as a dep either, btw).

Can we avoid relying on `six`?

We managed in EasyBuild, so we can manage here? See https://github.com/easybuilders/easybuild-framework/blob/develop/easybuild/tools/py2vs3/py2.py#L75